### PR TITLE
Installation instructions

### PR DESCRIPTION
**Pull request by scott-w**
Adds the installation instructions explicitly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,14 @@
...
 
+## Installing
+
+REST Framework LaTeX can be downloaded from PyPI:
+
+```bash
+pip install rest-framework-latex
+```
+
 [Documentation](http://drf-latex.readthedocs.io/en/latest/)
```
Comment (kennydude) [inline README.md]: This link should be moved upwards

Comment (scott-w) [inline README.md]: How do you mean?

Comment (scott-w) [inline README.md]: Oh you mean the documentation link?

Comment (kennydude) [inline README.md]: Yes

**Pull request merged**